tests/test_affect_history.py

```
@@ -0,0 +1,645 @@
+
+
+class TestAffectHistoryPrune:
+    """Test snapshot pruning logic."""
+    
+    def test_prune_by_age(self, sample_baseline):
+        """Test age-based pruning."""
+        history = AffectHistory(max_age_hours=0.001)  # ~3.6 seconds
+        values = np.array([0.5, 0.1, 0.1, 0.1, 0.5, 0.1, 0.4], dtype=np.float32)
+        
+        # Record snapshot
+        snapshot1 = history.record(values, sample_baseline, "old")
```

Ruff (0.15.0)

[error] Local variable `snapshot1` is assigned to but never used

Remove assignment to unused variable `snapshot1`

(F841)